docs/topics/braintree.rst

```diff
@@ -187,3 +181,86 @@ Information about the payment method is stored in solitude.
     :query braintree_buyer: the primary key of the braintree_buyer.
     :query braintree_buyer__buyer__uuid: the uuid for the buyer.
     :query active: the active status.
+
+.. _subscription-label:
+
+Subscription
+------------
+
+Creates a subscription in Braintree and the corresponding subscription in
+solitude.
+
+.. http:post:: /braintree/subscription/
+
+    :<json string paymethod: the uri of the payment method.
+    :<json string plan: the id of the plan being purchased.
+
```

Comment:
In other docs I liked how you linked this ``mozilla`` output to the section that explains it in detail. In this case I think it should link to the ``/braintree/mozilla/subscription/<id>`` doc section.